### PR TITLE
Update illustrations for x-axis date change

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3260](https://openedx.atlassian.net/browse/DOC-3260)

The format used for dates in Insights changed from MM/DD to DD Mon YYYY. This PR updates all images that show dates, and updates text when necessary to accommodate new or changed images.
### Date Needed

Apologies, but this may release at any time, so ASAP.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @thallada 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check/copy edit/dev edit): @pdesjardins or @catong 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: @dsjen 
### Testing
- [x] Ran make html without warnings or errors
### Post-review
- [x] Add description to release notes task as a comment
- [ ] Squash commits

### CODE REVIEW

**Comment by thallada** [comment]
This looks great :+1:

**Comment by stroilova** [comment]
👍  Thank you!

File: docs/en_us/dashboard/source/engagement/Engagement_Content.rst [hunk 1]
```diff
@@ -43,17 +43,16 @@ report.
 
 Examples of the Weekly Student Engagement chart follow. The first example shows
 a course that started only a few weeks ago. Callouts provide context for the
-data that is shown by indicating several recent occurrences: the beginning of
-the beta test, the course start date, and the due date of the first homework.
+data that is shown by indicating several recent occurrences: 1) course
+development on edx.org, 2) the beginning of the beta test, 3) the course start
+date, and 4) the due date of the first homework.
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/engagement/Engagement_Content.rst]: Why not an itemized or ordered list?
#.
#.
#.

File: docs/en_us/dashboard/source/learners/Learner_Activity.rst [hunk 2]
```diff
@@ -122,21 +122,22 @@ understanding of this learner's experience, you could go back to the learner
...
+     and just two days with problems correct activity.
 
 Knowing the context of how your course is set up, this pattern might indicate
-when the learner reached the first homework assignment. Or, it might indicate
-that the learner completed all of the ungraded practice questions and the
-midterm on one day.
+that the learner completed the first homework assignment in your self-paced
+course and then, after some weeks off, completed the second homework
+assignment. Or, it might indicate that the learner started your
+instructor-paced course on track, but is now rushing to complete the remaining
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/learners/Learner_Activity.rst]: on track > on schedule?

**Comment by pdesjardins** [comment]
Two minor comments, 👍

File: docs/en_us/dashboard/source/engagement/Engagement_Content.rst (hunk 1 above)
Comment (lamagnifica) [inline docs/en_us/dashboard/source/engagement/Engagement_Content.rst]: @pdesjardins I just jammed the numbers in (the old image had text overlaid on the image, and we're trying to replace those with numbered callouts, but I was rushing and didn't really think this part through). Will change.

**Pull request merged**